entrypoint.sh

```diff
@@ -101,12 +101,13 @@ case "$API_METHOD" in
       echo "Successful request: $RESPONSE_CODE"
       # Mask output secret
       if [ "$IS_FILE" == "true" ]; then
-         echo "::add-mask::$(cat response.txt)"
```

Comment:
I think we also need the source file to be masked too. Just in case it is printed anywhere in the caller workflows